src/LibLog/LogProviders/LibLogException.cs

```diff
@@ -1,11 +1,22 @@
 ﻿namespace YourRootNamespace.Logging.LogProviders
 {
     using System;
+#if LIBLOG_EXCLUDE_CODE_COVERAGE
+    using System.Diagnostics.CodeAnalysis;
+#endif
 
+#if LIBLOG_EXCLUDE_CODE_COVERAGE
+    [ExcludeFromCodeCoverage]
+#endif
+#if !LIBLOG_PROVIDERS_ONLY || LIBLOG_PUBLIC
     /// <summary>
     /// Exception thrown by LibLog.
     /// </summary>
-    public class LibLogException : Exception
+    public
+#else
+    internal
+#endif
+    class LibLogException : Exception
```

Comment:
Shouldn't it inherit from `InvalidOperationException`?

Reply:
@samcragg, author of #169 suggests that and it seems like a good idea.

Reply:
IMO the name of the exception (LibLogException) is too general for that. Also this would be a semantic change?

So I think keeping it like this is better.

Reply:
@mniak Well LibLog is now in palliative care. No need to disturb the patient more than needed.

Reply:
No problem. Carry on